packages/nc-gui/utils/dataUtils.ts

```
@@ -145,3 +146,284 @@
+        }
+      }
+    }
+
+    switch (filter.logical_op) {
+      case 'or':
+        isValid = isValid || !!res
+        break
+      case 'not':
+        isValid = isValid && !res
+        break
+      case 'and':
```

Biome

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)